security/crowdsec/pkg-descr

```diff
@@ -8,6 +8,12 @@ WWW: https://crowdsec.net/
 Plugin Changelog
 ================
 
+1.0
+
+* first non-devel release
+* changed service restart to reload on hub update; fixed "service oscrowdsec status"
+* note about ramfs in /var
```

Comment:
/var MFS is changed to /var/log MFS only so this may not apply

Reply:
well I can remove it. Good, I didn't like the scary warning